README.md

```diff
@@ -7,10 +7,22 @@ Static facades for using [TensorFlow](https://www.tensorflow.org/) in [ScalaPy](
 Currently contains minimal bindings for usage with TensorFlow.
 
 ## Installation
-`scalapy-tensorflow` is published on Maven Central, so you can add it to your project by including it as a dependency.
+To install `scalapy-tensorflow` use [JitPack](https://jitpack.io/)
+for scala 2.13:
+```
+scalaVersion := "2.13.3"
+
+resolvers in ThisBuild += "jitpack" at "https://jitpack.io"
+
+libraryDependencies += "com.github.VirtuslabRnD.scalapy-tensorflow" % "scalapy-tensorflow-jvm_2.13" % "43a784c8d74be8235bf4ecb29a766647a02ecb55"
```

Comment:
Hmm maybe sth like:
```suggestion
libraryDependencies += "com.github.VirtuslabRnD.scalapy-tensorflow" %% "scalapy-tensorflow-jvm" % "43a784c8d74be8235bf4ecb29a766647a02ecb55"
```
instead?

Reply:
Also, maybe instead of a hardcoded git commit hash... maybe sth like `develop-SNAPSHOT`

Reply:
Nice, I forgot about that.